### PR TITLE
Add USR Certificate Directory for sef-site

### LINKED ISSUE #1147: Add USR Certificate Directory for sef-site
**Is your feature request related to a problem? Please describe.**
Sef-site needs a directory to create a page for issuing USR digital certificates

**Describe the solution you'd like**
Create a directory in sef-site (usr) and integrate the API with template

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Madhawa97**
## Purpose
The purpose of this PR is to fix #1147 

## Goals
To create a page for issuing USR digital certificates

## Approach
Create a directory in sef-site named 'usr' and integrate the API with template

### Preview Link

https://pr-1148-sef-site.surge.sh/fellowship/usr/?id=2

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
java 17.0.1
macOS 12.1

### CODE REVIEW

File: usr/index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    </div>
+  </script>
+  <script>
+    $(function () {
+      let searchParams = new URLSearchParams(window.location.search);
+      if (searchParams.has('id')) {
+        let id = searchParams.get('id');
+        $.ajax({
+          type: 'get',
+          url: 'https://sef-strapi.herokuapp.com/api/usrs/' + id,
```
Comment (jayasanka-sack) [inline usr/index.html]: Shall we make the endpoint more readable?

File: usr/index.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
```
Comment (jayasanka-sack) [inline usr/index.html]: This file should come under `fellowship/usr`

Comment (Madhawa97) [reply usr/index.html]: will do

File: usr/index.html (hunk 1 above)
Comment (Madhawa97) [reply usr/index.html]: received the new link from @gajindubandara and it was updated with the last commit

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Madhawa97 !

Could you please have a discussion with the USR team and update the certificate content with their design?

**Comment by Madhawa97** [comment]
@jayasanka-sack Sure I will so that

**Comment by Madhawa97** [comment]
You're welcome. And Sure @jayasanka-sack , I will do that.

**Pull request merged**